packages/dev/docs/pages/react-spectrum/testing.mdx

```diff
@@ -337,6 +337,12 @@ fireEvent.mouseMove(thumb, {pageX: 50});
 fireEvent.mouseUp(thumb, {pageX: 50});
```

Comment:
at some point we should update this to use `userEvent.pointer` and not rely on mouse events. will be important as we update usePress.

Reply:
sounds good, I've made a note to do some when the usePress changes go in